Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3028,31 +3028,31 @@ public static InputActionAsset actions
         {
             get
             {
-                if (s_projectWideActions != null)
-                    return s_projectWideActions;
+                if (s_ProjectWideActions != null)
+                    return s_ProjectWideActions;
 
                 #if UNITY_EDITOR
-                s_projectWideActions = Editor.ProjectWideActionsAsset.GetOrCreate();
+                s_ProjectWideActions = ProjectWideActionsAsset.GetOrCreate();
                 #else
-                s_projectWideActions = Resources.FindObjectsOfTypeAll<InputActionAsset>().FirstOrDefault(o => o != null && o.name == kProjectWideActionsAssetName);
+                s_ProjectWideActions = Resources.FindObjectsOfTypeAll<InputActionAsset>().FirstOrDefault(o => o != null && o.name == kProjectWideActionsAssetName);
                 #endif
 
-                if (s_projectWideActions == null)
+                if (s_ProjectWideActions == null)
                     Debug.LogError($"Couldn't initialize project-wide input actions");
-                return s_projectWideActions;
+                return s_ProjectWideActions;
             }
 
             set
             {
                 if (value == null)
                     throw new ArgumentNullException(nameof(value));
 
-                if (s_projectWideActions == value)
+                if (s_ProjectWideActions == value)
                     return;
 
-                s_projectWideActions?.Disable();
-                s_projectWideActions = value;
-                s_projectWideActions.Enable();
+                s_ProjectWideActions?.Disable();
```

Comment:
I noticed there is a warning on these lines saying it disables Unity object lifetime checks to use ?. operator here. Consider maybe an if-statement instead?